tests/xbot/common/subsystems/drive/HeadingAssistModuleTest.java

```diff
@@ -0,0 +1,123 @@
+package xbot.common.subsystems.drive;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import xbot.common.injection.BaseWPITest;
+import xbot.common.subsystems.drive.control_logic.HeadingAssistModule;
+import xbot.common.subsystems.drive.control_logic.HeadingModule;
+
+public class HeadingAssistModuleTest extends BaseWPITest {
+
+    HeadingAssistModule ham;
+    
+    @Override
+    public void setUp() {
+        // TODO Auto-generated method stub
```

Comment:
remove

Reply:
Thanks. I need to turn on the Eclipse TODO finder.